compiler/src/pkg/manifest.rs

```diff
@@ -1,12 +1,50 @@
+use crate::config::Config;
 use crate::pkg::version::Version;
 use blake3;
 use std::fmt;
-use std::fs::{File, OpenOptions};
+use std::fs::{read_dir, File, OpenOptions};
 use std::io::{BufRead, BufReader, Read, Write};
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
 pub const MANIFEST_FILE: &str = "inko.pkg";
 
+pub fn all_manifests(
+    config: &Config,
+) -> Result<Vec<(Manifest, PathBuf)>, String> {
+    let mut manifests = Vec::new();
+    let mut paths = Vec::new();
+    let dependencies = &config.dependencies;
+
+    if let Ok(dependencies) = read_dir(dependencies) {
+        for dependency in dependencies.filter_map(Result::ok) {
+            if let Ok(versions) = read_dir(dependency.path()) {
+                for version in versions.filter_map(Result::ok) {
+                    let mut path = version.path();
+                    path.push(MANIFEST_FILE);
```

Comment:
I think this should work just as well:

```suggestion
                    let path = version.path().join(MANIFEST_FILE);

```